libs/ui/src/lib/marketplace/pages/wishlist/wishlist.component.ts

```diff
@@ -84,6 +84,14 @@ export class WishlistComponent implements OnInit, OnDestroy {
     });
   }
 
+  public clear(event: Event) {
+    event.stopPropagation();
```

Comment:
You don't need `stopPropagation` here 
```suggestion
  public clear() {
```